README.md

```diff
@@ -24,21 +21,62 @@ We're hiring great people to join our team in Copenhagen. Head over to [our jobs
 - `./playground`: source code of the playground
 - `./definitions.js`: constant definitions
 
-
 ## How can I help?
-You can help by:
+You can help by helping us find, add and improve our Life Cycle Assesment / Carbon footprint data.
+
```

Comment:
Consider adding one sentence:
"This repository also includes integrations to third party services [link to integrations folder]. Feel free to extend those or suggest new ones if an official API is available"

Reply:
I do not want contributors to add more B2C apis though.